packet_testing/ext-hdr-test-packets.py

```diff
@@ -69,12 +42,12 @@ def run(self):
             # packet using the fragment header
             fragment_packet = scapy.all.Ether() / ipv6_packet / scapy.all.IPv6ExtHdrFragment(offset=56059)
 
-            # Packets using RFC3692 - UDP, TCP, ICMP
-            ipv6_rfc3692_header = IPv6ExtHdrRFC3692_style_experiment(value=56059)
+            # Packets using Destination options - UDP, TCP, ICMP
+            ipv6_dest_opt_header = scapy.all.IPv6ExtHdrDestOpt(options=scapy.all.PadN(optdata='0'))
```

Comment:
I think we should document elsewhere the "behavior" we observed with the autopadding not working if one passes the `length` as a parameter. Not necessarily here, but should be somewhere.